### DISCUSSION TITLE
v1视频标题可以像youtube放在视频下方，v2还有这个设置吗

### DISCUSSION THREAD

**Discussion by bubbleboy11** [the1812/Bilibili-Evolved#3631] [提问 / Q&A] [answered]
rt

**Comment by Geeyun-JY3** [comment, marked as answer]
`播放器置顶` 就是。

不过
![image](https://user-images.githubusercontent.com/26504152/182055335-5e0e279f-701e-4cc6-9f01-17b6ac06674c.png)

**Comment by MaxHasBeenUsed** [comment]
如上所说，该功能目前已经停止维护，并无法于”新版播放页“正常工作。如果您跟我一样喜爱该功能，欢迎在[这个 discussion](https://github.com/the1812/Bilibili-Evolved/discussions/3582#discussion-4290858)下参与投票。
